static/app/views/insights/common/components/ModulePageFilterBar.tsx

```diff
@@ -0,0 +1,70 @@
+import {type ComponentProps, Fragment, useEffect, useState} from 'react';
+
+import {DatePageFilter} from 'sentry/components/organizations/datePageFilter';
+import {EnvironmentPageFilter} from 'sentry/components/organizations/environmentPageFilter';
+import PageFilterBar from 'sentry/components/organizations/pageFilterBar';
+import {ProjectPageFilter} from 'sentry/components/organizations/projectPageFilter';
+import {Tooltip} from 'sentry/components/tooltip';
+import {t} from 'sentry/locale';
+import {SECOND} from 'sentry/utils/formatters';
+import useProjects from 'sentry/utils/useProjects';
+import {useHasFirstSpan} from 'sentry/views/insights/common/queries/useHasFirstSpan';
+import type {ModuleName} from 'sentry/views/insights/types';
+
+type Props = {
+  moduleName: ModuleName;
+  extraFilters?: React.ReactNode;
+  onProjectChange?: ComponentProps<typeof ProjectPageFilter>['onChange'];
+};
+
+const CHANGE_PROJECT_TEXT = t('Make sure you have the correct project selected.');
+
+export function ModulePageFilterBar({moduleName, onProjectChange, extraFilters}: Props) {
```

Comment:
Heads up, you named the file `ModulePageFilterBar.tsx`, to be consistent with our codebase the first character has to be lowercase